libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -34,13 +36,17 @@ void ExtVotesPacketHandler::sendPbftVote(dev::p2p::NodeID const &peer_id, std::s
 }
 
 void ExtVotesPacketHandler::sendPbftNextVotes(dev::p2p::NodeID const &peer_id,
-                                              std::vector<std::shared_ptr<Vote>> const &send_next_votes_bundle) {
+                                              std::vector<std::shared_ptr<Vote>> const &send_next_votes_bundle,
+                                              uint32_t index) {
   if (send_next_votes_bundle.empty()) {
```

Comment:
I think here should be a check if the peer still exist (is connected)

Reply:
no longer applies